pkgs/desktops/plasma-5/kwin/default.nix

```
@@ -12,7 +12,9 @@
   kcoreaddons, kcrash, kdeclarative, kdecoration, kglobalaccel, ki18n,
   kiconthemes, kidletime, kinit, kio, knewstuff, knotifications, kpackage,
   kscreenlocker, kservice, kwayland, kwidgetsaddons, kwindowsystem, kxmlgui,
-  plasma-framework, libcap, libdrm, mesa
+  plasma-framework, libcap, libdrm, mesa,
+  
+  lowLatencyPatch ? true, fetchpatch ? null
```

```suggestion
  lowLatencyPatch ? true, fetchpatch
```

Should we enable this on default? I am not sure.

---

No, I think we should not add patches by default that aren't fixing some bug. But, we can add a top-level attribute for it, so it will be built with and without the patch on Hydra.